cli/paas/client.go

```diff
@@ -103,6 +118,20 @@ func (c *CarrierClient) CreateOrg(org string) error {
 
 // Delete deletes an app
 func (c *CarrierClient) Delete(app string) error {
+	_, err := c.giteaClient.DeleteRepo(c.config.Org, app)
+	if err != nil {
+		return errors.Wrap(err, "failed to delete repo")
+	}
+
+	c.ui.Normal().WithStringValue("name", app).Msg("Deleted app repo.")
+
+	err = c.eiriniClient.EiriniV1().LRPs(c.config.EiriniWorkloadsNamespace).Delete(context.Background(), app, metav1.DeleteOptions{})
+	if err != nil {
+		return errors.Wrap(err, "failed to delete eirini lrp")
+	}
+
+	c.ui.Normal().WithStringValue("name", app).Msg("Deleted app LRP.")
```

Comment:
Should we leak these interna (Repo and LRP) to the user ?
Maybe just tell that the application was deleted ?
In the failures I can see that it is useful to know which component made trouble.
In the success less so.

Reply:
Was thinking the same. I'll use more generic terms.